service/src/main/kotlin/fi/espoo/evaka/pis/PersonQueries.kt

```diff
@@ -90,6 +91,10 @@ fun Database.Transaction.createPerson(person: PersonIdentityRequest): PersonDTO
         .bind("email", person.email)
         .map(toPersonDTO)
         .first()
+
+    createMessageAccountForPerson(result.id)
```

Comment:
:man_shrugging:

Reply:
@kimifoud and I decided to have a separate "service layer", not a Spring service though, that contains this logic. I pushed a change where that logic is now in `Person.kt`, and all places that create persons were changed to call the functions from that file instead.

I didn't replicate all the database functions there, though, because that would have been overkill.